pkg/sql/logictest/logic.go

```diff
@@ -3311,30 +3306,8 @@ func (t *logicTest) execQuery(query logicQuery) error {
 	t.noticeBuffer = nil
 
 	db := t.db
-	var closeDB func()
 	if query.nodeIdx != t.nodeIdx {
-		var pgURL url.URL
-		if t.testserverCluster != nil {
-			pgURL = *t.testserverCluster.PGURLForNode(query.nodeIdx)
-			pgURL.User = url.User(t.user)
-			pgURL.Path = "test"
-		} else {
-			addr := t.cluster.Server(query.nodeIdx).ServingSQLAddr()
-			if len(t.tenantAddrs) > 0 {
-				addr = t.tenantAddrs[query.nodeIdx]
-			}
-			var cleanupFunc func()
-			pgURL, cleanupFunc = sqlutils.PGUrl(t.rootT, addr, "TestLogic", url.User(t.user))
-			defer cleanupFunc()
-			pgURL.Path = "test"
-		}
-
-		db = t.openDB(pgURL)
-		closeDB = func() {
```

Comment:
is this closeDB no longer needed?

Reply:
Yeah, it's not required any more because now that we're keeping track of these clients in `t.clients`, they'll be be closed during cluster shutdown here: https://github.com/cockroachdb/cockroach/blob/3f73cfbcab26521fdcc5a379828d1ea2db7f4aa0/pkg/sql/logictest/logic.go#L1779-L1783